blog/2023-11-20-stately-studio-2-0/index.mdx

```diff
@@ -0,0 +1,146 @@
+
+When it comes to documenting your app logic, states and transitions aren’t the whole story. You often want to include use cases, features, workflows, and more. One of our most frequent requests is from teams who want to tie their logic to real-life user interfaces. Assets provide a way to connect user interface design to logic in a way your whole team can understand.
+
+In the future, you’ll have the option to incorporate your assets into the generated UI code. Stately AI will help summarize and improve your state machines based on the contextual information in the assets you’ve provided.
+
+Are you an enterprise team looking to store your assets on your own cloud or servers? [Contact us](mailto:support@stately.ai) for more information about Stately Enterprise.
+
+## Stately Sky
+
+At [Stately.ai](http://stately.ai/), we want to help you and your team use state machines to improve every aspect of your software development process, from design to development to testing to deployment to monitoring. Last week, we launched **Stately Sky** for our Pro users; if you use state machines for workflows, you can now use Stately Sky to deploy your state machines to the cloud and get all the benefits of serverless computing.
+
+[image]
+
+With Stately AI and Stately Sky together, you can go from an idea to a deployed workflow in minutes. The Stately editor helps you create much more flexible workflows than other workflow tools—you can use these workflows on the front end, back end, or both. Perfect for onboarding flows, collaborative apps, games, AI agents, and more. Stately Sky is also backed by Cloudflare Workers so that you can run your state machines in 200+ locations worldwide, with low latency and high availability. 
+
+You can [read more about Stately Sky in Gavin’s blog post](https://stately.ai/blog/2023-11-13-introducing-stately-sky).
+
+## Enhanced Stately editor
+
+Among all these huge new features, we have updated the Stately editor’s UI. We want [Stately.ai](http://stately.ai/) to feel as natural as a white-boarding or diagramming tool, while giving you the power to create robust state machines at any level of complexity.
+
+These updates significantly improve the editor’s performance, especially when working with large machines. We’ve also streamlined the editor with reorganized panels and a redesigned header and footer. The new design makes accessing the tools you need faster while also providing information about your machine, including its locked status, project visibility, and current version.
+
+### Targetless transitions
+
+As a part of our improved support for XState in the Stately editor, the editor now supports targetless transitions. Targetless transitions are useful when you want a transition on your root state but don’t want to keep resetting your machine to the initial state. They also open up the possibility for multiple target state support in the future, which is handy if you want to transition to simultaneous parallel states.
```

Comment:
Should we mention something about being able to trigger actions without any actual state change at all?